### DISCUSSION TITLE
Create the connection to sleep 1 second and then query, the speed becomes very slow

### DISCUSSION THREAD

**Discussion by zyy20191** [milvus-io/milvus#18210] [Q&A and General discussion] [answered]
Steps:
1. Creating a connection
2. time.sleep(1)
3. Query operation

Step 1, 2, 3 is 20ms slower than step 1, 3

**Comment by yhmo** [comment, marked as answer]
I test the steps, it is 15ms gap on my machine. All the additional time cost is at rpc level, I guess it is caused by grpc internal machinery.

**Reply by zyy20191** [reply]
Can I set grpc to reduce latency?

**Reply by yhmo** [reply]
> Can I set grpc to reduce latency?

The grpc setting is not exposed in pymilvus. And I have no idea how to optimize grpc in such case.
